.zenodo.json

```
@@ -6,7 +6,126 @@
     "brain imaging data structure",
     "neuroscience",
     "neuroimaging",
-    "neuroinformatics",
+    "neuroinformatics"
+  ],
+  "creators": [
+    {
+      "name": "Blair, Ross"
+    },
+    {
+      "name": "Michael, Zack"
+    },
+    {
+      "name": "Gorgolewski, Krzysztof J."
+    },
+    {
+      "name": "Hardcastle, Nell"
+    },
+    {
+      "name": "Hobson-Lowther, Teal"
+    },
+    {
+      "name": "Nishikawa, David"
+    },
+    {
+      "name": "Bhogawar, Suyash"
+    },
+    {
+      "name": "Appelhoff, Stefan"
+    },
+    {
+      "name": "Jas, Mainak"
+    },
+    {
+      "name": "Grass, Brian"
+    },
+    {
+      "name": "Holdgraf, Chris"
+    },
+    {
+      "name": "Jones, Alexander"
+    },
+    {
+      "name": "Goyal, Rohan"
+    },
+    {
+      "name": "Markiewicz, Christopher J."
+    },
+    {
+      "name": "Oostenveld, Robert"
+    },
+    {
+      "name": "Noack, Gregory"
+    },
+    {
+      "name": "Triplett, William"
+    },
+    {
+      "name": "Naveau, Mika\u00ebl"
```

```suggestion
      "name": "Naveau, Mika\u00ebl",
      "affiliation": "CYCERON imaging plateform, Caen, France",
      "orcid": "https://orcid.org/0000-0002-4685-0057"
```

@effigies : thanks for the ping !